java/java.source.base/src/org/netbeans/modules/java/source/parsing/JavacParser.java

```diff
@@ -181,8 +182,8 @@
     private FileObject root;
     //ClassPaths used by the parser
     private ClasspathInfo cpInfo;
-    //Count of files the parser was created for
-    private final int sourceCount;
+    //XXX: Count of files the parser was created for
```

Comment:
Would probably be good to cleanup the comment here (i.e. adjust it to the current state).